### PR TITLE
adding pypi badges to repository nereid #6825

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+nereid-catalog-tree
+===================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid.svg?branch=3.2)](https://travis-ci.org/openlabs/nereid)
```
Comment (aroraumang) [inline README.md]: This should be for develop branch.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+Warning: This code has been merged with nereid
+==============================================
+[![Build Status](https://travis-ci.org/openlabs/nereid.svg?branch=develop)](https://travis-ci.org/openlabs/nereid)
```
Comment (PritishC) [inline README.md:5]: Add **one** blank line as explained in another PR.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Warning: This code has been merged with nereid
+==============================================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid.svg?branch=develop)](https://travis-ci.org/openlabs/nereid)
+[![Downloads](https://pypip.in/download/trytond_nereid/badge.svg)](https://pypi.python.org/pypi/trytond_nereid/)
+[![Latest Version](https://pypip.in/version/trytond_nereid/badge.svg)](https://pypi.python.org/pypi/trytond_nereid/)
+[![Development Status](https://pypip.in/status/trytond_nereid/badge.svg)](https://pypi.python.org/pypi/trytond_nereid/)
+[![Coverage Status](https://coveralls.io/repos/openlabs/nereid/badge.svg?branch=develop)](https://coveralls.io/r/openlabs/nereid?branch=develop)
+
+This code is now a part of [Nereid] (http://nereid.openlabs.co.in)_
```
Comment (PritishC) [inline README.md]: Underscore at the end is not required.

**Comment by PritishC** [comment]
Add task ID to PR title.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
+Warning: This code has been merged with nereid
+==============================================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid.svg?branch=develop)](https://travis-ci.org/openlabs/nereid)
```
Comment (PritishC) [inline README.md]: trytond-nereid

File: README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+[![Downloads](https://pypip.in/download/trytond_nereid/badge.svg)](https://pypi.python.org/pypi/trytond_nereid/)
+[![Latest Version](https://pypip.in/version/trytond_nereid/badge.svg)](https://pypi.python.org/pypi/trytond_nereid/)
+[![Development Status](https://pypip.in/status/trytond_nereid/badge.svg)](https://pypi.python.org/pypi/trytond_nereid/)
+[![Coverage Status](https://coveralls.io/repos/openlabs/nereid/badge.svg?branch=develop)](https://coveralls.io/r/openlabs/nereid?branch=develop)
```
Comment (PritishC) [inline README.md]: trytond-nereid

**Comment by PritishC** [comment]
LGTM.

**Pull request closed without merging**